### ISSUE TITLE
lisp ftplugin: quadruple semicolon missing from comments setting

### ISSUE THREAD

**Issue by cwfoo** [vim/vim#7878]
**Describe the bug**
Quadruple semicolon `;;;;` (ref: [Common Lisp](http://www.lispworks.com/documentation/HyperSpec/Body/02_ddbd.htm), [Emacs Lisp](https://www.gnu.org/software/emacs/manual/html_node/elisp/Comment-Tips.html)) is missing from ftplugin/lisp.vim's `comments` setting:

https://github.com/vim/vim/blob/ada1d870b4a818151cfba1c18962af2369b88df9/runtime/ftplugin/lisp.vim#L17-L24

This affects the joining of comment lines using `J` when `set formatoptions+=j`.

**To Reproduce**
1. Open a new lisp file (e.g. `vim myprogram.lisp`).
2. `set formatoptions+=j`
3. Enter the following starting on the first line:
    ```lisp
    ;;;; First line.
    ;;;; Second line.
    ```
4. Position the cursor on the first line.
5. In normal mode, press `J`.
6. Result:
    ```lisp
    ;;;; First line.  ; Second line.
    ```
   Notice how only three semicolons have been removed.

**Expected behavior**
Expected result:
```lisp
;;;; First line.  Second line.
```

**Environment**
 - Vim version: 8.1.2269
 - ftplugin/lisp.vim version: Oct 23, 2013
 - OS: Ubuntu 20.04
 - Terminal: GNOME Terminal

**Additional context**
Proposed solution (copied from [ftplugin/scheme.vim](https://github.com/vim/vim/blob/v8.2.2532/runtime/ftplugin/scheme.vim#L17)):

```diff
diff --git a/runtime/ftplugin/lisp.vim b/runtime/ftplugin/lisp.vim
index 130f30b14..9825d2484 100644
--- a/runtime/ftplugin/lisp.vim
+++ b/runtime/ftplugin/lisp.vim
@@ -14,13 +14,11 @@ endif
 " Don't load another plugin for this buffer
 let b:did_ftplugin = 1
 
-setl comments=:;
+setl comments=:;;;;,:;;;,:;;,:;,sr:#\|,mb:\|,ex:\|#
 setl define=^\\s*(def\\k*
 setl formatoptions-=t
 setl iskeyword+=+,-,*,/,%,<,=,>,:,$,?,!,@-@,94
 setl lisp
 setl commentstring=;%s
 
-setl comments^=:;;;,:;;,sr:#\|,mb:\|,ex:\|#
-
 let b:undo_ftplugin = "setlocal comments< define< formatoptions< iskeyword< lisp< commentstring<"
```

cc: @khorser (maintainer).

**Comment by brammool** [comment]
Chuan Wei Foo wrote:

> **Describe the bug**
> [Quadruple semicolon](http://www.lispworks.com/documentation/HyperSpec/Body/02_ddbd.htm) is missing from ftplugin/lisp.vim's `comments` setting:
> 
> https://github.com/vim/vim/blob/ada1d870b4a818151cfba1c18962af2369b88df9/runtime/ftplugin/lisp.vim#L17-L24
> 
> This affects the joining of comment lines using `J` when `set formatoptions+=j`.
> 
> **To Reproduce**
> 1. Open a new lisp file (e.g. `vim myprogram.lisp`).
> 2. `set formatoptions+=j`
> 3. Enter the following starting on the first line:
>     ```lisp
>     ;;;; First line.
>     ;;;; Second line.
>     ```
> 4. Position the cursor on the first line.
> 5. In normal mode, press `J`.
> 6. Result:
>     ```lisp
>     ;;;; First line.  ; Second line.
>     ```
>    Notice how only three semicolons have been removed.
> 
> **Expected behavior**
> Expected result:
> ```lisp
> ;;;; First line.  Second line.
> ```
> 
> **Environment**
>  - Vim version: 8.1.2269
>  - ftplugin/lisp.vim version: Oct 23, 2013
>  - OS: Ubuntu 20.04
>  - Terminal: GNOME Terminal
> 
> **Additional context**
> Proposed solution:
> 
> ```diff
> diff --git a/runtime/ftplugin/lisp.vim b/runtime/ftplugin/lisp.vim
> index 130f30b14..9825d2484 100644
> --- a/runtime/ftplugin/lisp.vim
> +++ b/runtime/ftplugin/lisp.vim
> @@ -14,13 +14,11 @@ endif
>  " Don't load another plugin for this buffer
>  let b:did_ftplugin = 1
>  
> -setl comments=:;
> +setl comments=:;;;;,:;;;,:;;,:;,sr:#\|,mb:\|,ex:\|#
>  setl define=^\\s*(def\\k*
>  setl formatoptions-=t
>  setl iskeyword+=+,-,*,/,%,<,=,>,:,$,?,!,@-@,94
>  setl lisp
>  setl commentstring=;%s
>  
> -setl comments^=:;;;,:;;,sr:#\|,mb:\|,ex:\|#
> -
>  let b:undo_ftplugin = "setlocal comments< define< formatoptions< iskeyword< lisp< commentstring<"
> ```
> 
> cc: @khorser (maintainer).

I haven't heard from Sergey for a long time, but let's see if we get a
response.

-- 
Why isn't there mouse-flavored cat food?

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by cwfoo** [comment]
Fixed in 9faec4e3d439968e21ad74e917aebb289df8f849, but the commit did not update the "Last Change:" date in the file.

The "Last Change:" date was later updated in dad4473f02e1fec86d43a2fc094536a4b27d3b25.
